example/profile.php

```diff
@@ -1,8 +1,11 @@
 <?php
-require_once './vendor/autoload.php'; // Make sure you run composer update inside the example folder before trying them
+
+// Make sure you run composer update inside the example folder before trying them
+require_once './vendor/autoload.php';
+// Error message
 $errorMsg = '';
 // Get and sanitize the token
-$token = (isset($_GET['token']))? preg_replace('/[^-a-zA-Z0-9_]/', '', $_GET['token']) : '';
+$token = isset($_GET['token']) ? preg_replace('/[^-a-zA-Z0-9_]/', '', $_GET['token']) : '';
```

Comment:
Do we need this step? `preg_replace('/[^-a-zA-Z0-9_]/'`

Reply:
I think we need it, it's sanitising the token, it's removing special characters from the token.

Reply:
Shouldn't the SDK just decode what it receives from Connect?

Reply:
Yes it should indeed, but we are trying to prevent any injection in case someone tries to modify the token because it's not a Post but a Get. We are just trying to show people how to make their code robust.
I am happy to remove it you still think it's not needed.